sky/clouds/aws.py

```diff
@@ -110,15 +110,14 @@ def get_default_ami(cls, region_name: str, instance_type: str) -> str:
                 }
                 assert region_name in amis, region_name
                 return amis[region_name]
-        # Deep Learning AMI GPU PyTorch 1.10.0 (Ubuntu 20.04) 20220308
+        # Deep Learning AMI GPU PyTorch 1.12.1 (Ubuntu 20.04) 20221025
         # https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;v=3;search=:64,:Ubuntu%2020,:Deep%20Learning%20AMI%20GPU%20PyTorch # pylint: disable=line-too-long
-        # Nvidia driver: 510.47.03, CUDA Version: 11.6
+        # Nvidia driver: 510.47.03, CUDA Version: 11.6 (supports torch==1.13.0+cu117)
```

Comment:
Q: Just to understand, if the old image also comes with CUDA 11.6, this image upgrade is not necessary to fix the pytorch issue? 

(I'm weary of us frequently updating images. If there's no strong reasons or user demands, I think keeping the images stable is better.)

Reply:
Good point!

The latest version of the AMI supports the PyTorch with the CUDA 11.7, which is the one installed by default with `pip install torch`, but the previous version does not.

I agree that we should not frequently update the images, and since the other cloud does not support PyTorch with CUDA 11.7. I am now leaning toward reverting it.

Reply:
have you checked if the AMI has an older version of CUDA under /usr/local (or other places)? then we only need to softlink /usr/local/cuda to the older version. this is how you are supposed to handle backwards compatibility

Reply:
e.g., in setup we can have `sudo rm /usr/local/cuda && sudo ln -s /usr/local/cudaxxxx /usr/local/cuda`

Reply:
I think the backward compatibility of cuda is handled well even without re-do the soft link, as the `pip install torch==1.12.1+cu113` works well on the VM with cuda 11.6 installed.